src/CBTListScreen.tsx

```diff
@@ -111,7 +116,13 @@ const ThoughtItemList = ({
   }
 
   const items = groups.map(group => {
-    const thoughts = group.thoughts.map(thought => (
+    const data = group.thoughts.filter(el => {
+      const itemText = `${el.alternativeThought} ${el.alternativeThought} ${el.challenge}`.toLowerCase()
```

Comment:
This is really clever. 👌 

But I think you meant to have `automaticThought` for one of them. 😅